bgcval2/analysis_compare.py

```diff
@@ -1566,7 +1566,8 @@ def areatotal(nc, keys):
                 print(
                     "analysis-Timeseries.py:\tWARNING:\tmodel files are not found:",
                     av[name]['modelFiles'], jobID)
-                if strictFileCheck: assert 0
+                if strictFileCheck: 
```

Comment:
@valeriupredoi I made some changes to strictFileCheck here. and below.

Reply:
cheers, I polished it for you, have a look! ideally this should sit outside the analysis module, so we can plug some integrity checks too - OK for now tho